website/src/react-native/showcase.js

```diff
@@ -329,6 +329,14 @@ var featured = [
     infoLink: 'https://engineering.huiseoul.com/building-a-conversational-e-commerce-app-in-6-weeks-with-react-native-c35d46637e07',
     infoTitle: 'Building a conversational E-commerce app in 6 weeks with React Native',
   },
+  {
+    name: 'Baidu(手机百度)',
+    icon: 'https://ss0.bdstatic.com/-0U0bnSm1A5BphGlnYG/tam-ogel/8b8eb88700776fe293f4f9a74a73aa18_121_121.jpg',
+    linkPlayStore: 'http://shouji.baidu.com/software/9896302.html',
+    linkAppStore: 'https://itunes.apple.com/en/app/shou-ji-bai-du-hai-liang-xin/id382201985?l=en&mt=8',
+    infoLink: 'http://baike.baidu.com/link?url=TW8YhcVN4tO_Jz5VqMclCjGhf12EEqMD_TeVC6efe2REZlx80r6T0dX96hdmNl36XogLyExXzrvFU9rFeqxg_K',
+    infoTitle: 'Baidu is a Search Engine APP that has been used by 600 millions users.',
```

Comment:
Just a grammatical nitpick: could you rephrase this sentence as:

"Baidu is a search engine that has 600 million users."